pkg/controllers/certmanager/controller.go

```diff
@@ -71,7 +71,7 @@ func (m *controller) validateCerts() error {
 	valid, err := m.renewer.ValidCert()
 	if err != nil {
 		logger.Error(err, "failed to validate cert")
```

Comment:
Can we move this error log after `apierrors.IsNotFound` check?

Reply:
Done.